code/game/turfs/simulated/wall_natural.dm

```diff
@@ -0,0 +1,161 @@
+var/list/strata_by_z = list()
+var/list/strata_material_by_z = list()
+var/list/natural_walls = list()
+/turf/simulated/wall/natural
+	name = "wall"
+	material = null
+	reinf_material = null
+	girder_material = null
+	construction_stage = -1
+	floor_type = /turf/simulated/floor/asteroid
+	var/strata
+	var/image/ore_overlay
+
+/turf/simulated/wall/natural/get_paint_examine_message()
+	. = SPAN_NOTICE("It has been <font color = '[paint_color]'>noticeably discoloured</font> by the elements.")
+
+/turf/simulated/wall/natural/Initialize()
+	if(!material)
+		if(!strata)
+			if(!global.strata_by_z["z"])
+				global.strata_by_z["z"] = pick(/decl/strata/sedimentary, /decl/strata/metamorphic, /decl/strata/igneous)
```

Comment:
Feel like this should be a var on strata (in the running to be a stata_by_z of a z level) and then  you filter them here.

Reply:
Would prefer to do that in a future PR, this is as complex as it needs to be for this PR and I am getting sick of conflicts with my other PRs.

Reply:
Implemented barebones version.